en/dev_setup/building_px4.md

```diff
@@ -97,8 +97,8 @@ The following list shows the build commands for common boards:
 * [Pixhawk 3 Pro](../flight_controller/pixhawk3_pro.md): `make px4_fmu-v4pro_default`
 * [Pixhawk Mini](../flight_controller/pixhawk_mini.md): `make px4_fmu-v3_default`
 * [Cube Black](../flight_controller/pixhawk-2.md): `make px4_fmu-v3_default`
-* Cube Yellow: `make hex_cube-yellow`
-* Cube Orange: `make hex_cube-orange`
+* Cube Yellow: `make cubepilot_cubeyellow_default`
```

Comment:
Thanks. Can you for these please also link to the associated autopilot pages as has been done for all the other ones?

Reply:
Good call. Will do.

Reply:
Thanks. Also please resolve the build conflicts.